src/transaction/types/input.rs

```diff
@@ -234,6 +235,8 @@ impl Input {
         }
     }
 
+    // TODO: Maybe we need to move `ResourceId` from `fuel-core` into `fuel-tx`(or `fuel-type`) and
```

Comment:
If not, I will remove this TODO, if yes, I will create an issue=)

Reply:
That depends on whether or not the resource id is going to be an attribute of the transaction. If yes, then it should be moved here